cc/client/client_test.cc

```diff
@@ -55,25 +55,24 @@ constexpr uint8_t kTestSessionSize = 8;
 class TestTransport : public TransportWrapper {
  public:
   static absl::StatusOr<std::unique_ptr<TestTransport>> Create() {
-    auto encryption_key_provider = EncryptionKeyProvider::Create();
-    if (!encryption_key_provider.ok()) {
-      return encryption_key_provider.status();
+    auto encryption_key = EncryptionKeyProvider::Create();
```

Comment:
Not sure I understand the rename: creating an `EncryptionKeyProvider` now returns an encryption key?

Reply:
I'm currently trying to distinguish between key handles and actual keys.

And the "provider" name will be dropped in a following PR